packages/kbn-apm-synthtrace/src/scenarios/mobile.ts

```diff
@@ -418,7 +434,23 @@ const scenario: Scenario<ApmFields> = async ({ scenarioOpts, logger }) => {
         });
       };
 
-      return [...androidDevices, ...iOSDevices].map((device) => sessionTransactions(device));
+      const appLaunchMetrics = (device: MobileDevice) => {
+        return clickRate.generator((timestamp, index) =>
```

Comment:
@yngrdyn was this suggested by Alex?

Reply:
No, it was all me 😅
It doesn't make sense to generate the launch metric when a user clicks, how can we address this separately from clickRate? Would it makes sense to attach this to the `range` itself?

Reply:
Created [issue](https://github.com/elastic/kibana/issues/155594) to look into it, although it might not be relevant as application launch count would be retrieved from lifecycle events no metrics